src/api/spec/helpers/webui/notification_helper_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Webui::NotificationHelper, type: :helper do
```

Comment:
Those specs aren't testing much. I wonder if testing this helper is worth it. There isn't much logic beside the two ternary operators. The rest of the code relies on Rails methods which are covered upstream.

Testing the helper to check the type, title and icon could be done. What do you think?

Reply:
Good idea. I've added additional specs for title and icon :slightly_smiling_face: